activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+*   Add support for hash and url configs in database hash of `ActiveRecord::Base.connected_to`.
```

Comment:
Great addition. Thank you! Just want to note that we don't add changelog entries about complementing features that haven't been released.